src/pages/jeongaYeon/Main/Feed.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from 'react';
+import Comment from './Comment';
+
+const Feed = ({ feed }) => {
+  const [value, setValue] = useState('');
+  const [comments, setComments] = useState([]);
+
+  const handleComment = event => {
+    const comment = event.target.value;
+    if (comment.length <= 0) return;
+    setValue(comment);
+    if (event.keyCode === 13) {
+      setComments(current => [...current, comment]);
+      setValue('');
+    }
+  };
+
+  const handleCommentBtn = () => {
+    if (value <= 0) return;
+    setComments(current => [...current, value]);
+    setValue('');
+  };
+
+  const handleCommentInput = event => {
+    setValue(event.target.value);
+  };
+
+  return (
+    <article className="feedsWrapper">
```

Comment:
```suggestion
    <article className="feed">
```
컴포넌트의 이름과 동일하게 하려면 이렇게 하는 게 맞겠네요!

Reply:
수정 완료했습니다!!